frontend/src/components/UserDetail/UserDetail.tsx

```diff
@@ -68,8 +70,21 @@ const UserDetail = ({
 }: UserDetailProps) => {
   const fullName = `${firstName} ${lastName}`;
   const [isShowing, setIsShowing] = useState(false);
+  const { refreshUser } = useContext(AuthContext);
+  const [showingToast, setToast] = useState(false);
+  const [showingToastDelete, setToastDelete] = useState(false);
   const { withDefaults } = useReq();
   const { refreshRiders } = useRiders();
+  const [confirmationModalisOpen, setConfirmationModalisOpen] = useState(false);
+
+  const openConfirmationModal = () => {
+    setConfirmationModalisOpen(true);
+  };
+
+  const closeConfirmationModal = () => {
+    setToastDelete(true);
```

Comment:
when you press cancel, the "The student has been deleted." still shows

Reply:
maybe with the new toast stuff, you dont need to have the "The student has been deleted." toast stuff in this component?